docs/source/using/mounting-host-folders.adoc

```diff
@@ -0,0 +1,122 @@
+[[mounting-host-folders]]
+= Mounting host folders
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[mount-host-folders-overview]]
+== Overview
+
+Some drivers mount a host folder within the VM so that you can share files between the VM and the host.
+These folders are not currently configurable, and are different for each driver and OS that you use.
+
+[cols=",,,",options="header",]
+|========================================
+|Driver |OS |HostFolder |VM
+|Virtualbox |Linux |/home |/hosthome
+|Virtualbox |OSX |/Users |/Users
+|Virtualbox |Windows |C://Users |/c/Users
+|VMWare Fusion |OSX |/Users |/Users
+|Xhyve |OSX |/Users |/Users
+|========================================
+
+NOTE: Host folder sharing is not implemented in the KVM and Hyper-V drivers. You can
+link:#mounting-custom-shared-folders[mount a CIFS-based shared folder] inside the VM instead.
+
+[[mounting-custom-shared-folders]]
+== Mounting custom shared folders
+
+The Minishift https://github.com/minishift/minishift-b2d-iso[Boot2Docker] ISO and the
+Minishift https://github.com/minishift/minishift-centos-iso[CentOS] ISO include `cifs-utils`
+and `fuse-sshfs`. These tools allow you to mount host folders using
+https://en.wikipedia.org/wiki/Server_Message_Block[CIFS]
+or https://en.wikipedia.org/wiki/SSHFS[SSHFS].
+
+NOTE: When you use the Boot2Docker ISO along with the VirtualBox driver, VirtualBox
+guest additions are automatically enabled and occupy the _/Users_ mountpoint as
+shown in the following examples. In this case you need to use a different mountpoint.
+
+[[cifs-folder-mount]]
+=== Mounting folders with CIFS
+
+In this example, you use CIFS-based sharing to mount `C:\Users` on a Windows host
+into the Minishift VM. On Windows 10, the `C:\Users` folder is shared by default
+and only needs locally-authenticated users.
+
+.  Find the local IP address from the same network segment as your
+Minishift instance.
++
+----
+$ Get-NetIPAddress | Format-Table`
+----
+
+.  Create a mountpoint and mount the shared folder.
++
+[source,sh]
```

Comment:
The issue isn't with AsciiDoctor, it's with the toolchains that we use downstream (and also upstream via AsciiBinder), and we simply don't know how those tools will respond to tags that are not defined in the guidelines. Granted, the guidelines might not be inclusive of all the supported tags, but if we want to use tags that are not listed we should consult the OpenShift writers and verify that these tags won't break anything throughout all the toolchains that we use. 

@LalatenduMohanty there is no conversion planned for downstream docs, this is the whole point of the convergence initiative, to be able to reuse content both in upstream and downstream, which is one of the reasons that following conventions and communicating whenever we want to establish/extend/modify the conventions is critical.

Bottom line - I personally don't see the value in adding extra tags to code blocks, but if we want to implement something like this we should discuss it first internally, then with openshift and possibly also the content services tooling team, and document the conventions so that we don't have to remember who decided what when and why.